inference/core/interfaces/http/http_api.py

```diff
@@ -943,6 +945,17 @@ async def describe_workflows_blocks(
                     dynamic_blocks_definitions=dynamic_blocks_definitions
                 )
 
+            @app.get(
+                "/workflows/blocks/schema",
```

Comment:
should the path be `/workflows/spec/schema` or similar since the schema is for the spec rather than blocks per se

Reply:
also need to add the new endpoint to the API gateway for this to work with hosted workflows.

This needs to be done in roboflow-infra, I think in this file:  https://github.com/roboflow/roboflow-infra/blob/1ab773c66633dbc0311443200877d5768fd0ef61/aws/serverless/modules/containerlambda/main.tf#L110

Reply:
We call the JSON workflow definition, so maybe `GET /workflows/definition/schema`

Reply:
Renaming to `/workflows/definition/schema`